### PR TITLE
rel notes

### PR DESCRIPTION
**Pull request by Amrita42**
The scope of the PR to to inform users of bug fixes, known issues, features/enhancements, and deprecated features for the 6.0.0 rel version of wmco.
- Applies to 4.11 only
@mburke5678

### CODE REVIEW

**Comment by Amrita42** [comment]
@jeana-redhat ptal

File: windows_containers/windows-containers-release-notes-6-x.adoc [hunk 1]
```diff
@@ -31,23 +31,35 @@ endif::openshift-origin[]
 [id="wmco-6-0-0"]
 == Release notes for Red Hat Windows Machine Config Operator 6.0.0
 
-This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in
+This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in link:https://access.redhat.com/errata/RHBA-2022:97930[RHBA-2022:97930].
```
Comment (jeana-redhat) [inline windows_containers/windows-containers-release-notes-6-x.adoc:34]: (We have discussed in Slack but noting the Live ID issue here)

File: windows_containers/windows-containers-release-notes-6-x.adoc [hunk 2]
```diff
@@ -31,23 +31,35 @@ endif::openshift-origin[]
...
+====
+
+=== Bug fixes
+* Previously, a Windows instance could not be unconfigured when the node's external IP was present without a (pointer record (PTR). As a result, the node's external IP address without reverse lookup records was used to associate the Windows instance. With this release, a Windows instance can be unconfigured when the node's external IP address is present without a PTR .
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2070892[**BZ#2070892**])
+
+=== Known issues
+* Windows nodes do not support pulling container images from secure private registries. Use images from public registries or pre-pull the images in the VM image.
+
+=== Deprecated feature
```
Comment (jeana-redhat) [inline windows_containers/windows-containers-release-notes-6-x.adoc:48]: These section titles are plural by convention even when it's just one. 

```suggestion
=== Deprecated features
```

File: windows_containers/windows-containers-release-notes-6-x.adoc [hunk 3]
```diff
@@ -31,23 +31,35 @@ endif::openshift-origin[]
...
+Because the Docker runtime is deprecated in Kubernetes 1.24, containerd is now the default runtime for WMCO-supported Windows nodes. Upon the installation of or an upgrade to WMCO 6.0.0, containerd is installed as a Windows service. The kubelet now uses containerd for image pulls instead of the Docker runtime. Users no longer need to enable the Docker-formatted container runtime or install the Docker container runtime on Bring-Your-Own-Host (BYOH) instances. You can continue to use nodes based on VM images that use Docker. The containerd runtime can run along with the Docker service.
 
-The WMCO supports a Windows golden image with or without Docker for vSphere and Bring-Your-Own-Host (BYOH) Windows instances. 
+The WMCO supports a Windows golden image with or without Docker for vSphere and Bring-Your-Own-Host (BYOH) Windows instances.
```
Comment (jeana-redhat) [inline windows_containers/windows-containers-release-notes-6-x.adoc:62]: Already defined above :) 

```suggestion
The WMCO supports a Windows golden image with or without Docker for vSphere and BYOH Windows instances.
```

**Review by jeana-redhat — COMMENTED** [review]
Just a couple tiny nits you can get when you get the Live ID :)

File: windows_containers/windows-containers-release-notes-6-x.adoc (hunk 3 above)
Comment (Amrita42) [reply windows_containers/windows-containers-release-notes-6-x.adoc:62]: Good catch :)

**Comment by Amrita42** [comment]
Thank you @jeana-redhat :+1:  for your timely guidance.

File: windows_containers/windows-containers-release-notes-6-x.adoc (hunk 1 above)
Comment (Amrita42) [reply windows_containers/windows-containers-release-notes-6-x.adoc:34]: so now we have a live ID :)

**Pull request merged**